### PR TITLE
Add class ParseHooks

### PR DESCRIPTION
**Pull request by phelipealves**

### CODE REVIEW

**Comment by phelipealves** [comment]
@gfosco

File: src/Parse/ParseHooks.php [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+            throw new ParseException('Hooks triggers not found.', 101);
+        }
+
+        return $result['results'];
+    }
+
+    /**
+     * Fetch a single cloud trigger.
+     *
+     * @param $className
```
Comment (GrahamCampbell) [inline src/Parse/ParseHooks.php]: param annotations without types are invalid

**Comment by gfosco** [comment]
:+1: Nice work, thanks for contributing :)

File: src/Parse/ParseHooks.php (hunk 1 above)
Comment (gfosco) [inline src/Parse/ParseHooks.php]: Ah I just merged, heh.  @phelipealves please send another pr addressing anything Graham uncovers? thanks y'all. :+1:

**Comment by phelipealves** [comment]
thanks GrahamCampbell
